modules/atom-ide-ui/pkg/atom-ide-outline-view/lib/OutlineView.js

```diff
@@ -468,6 +470,19 @@ function renderTextToken(
   );
 }
 
+function getTooltipTitle(
+  outline: OutlineTreeForUi,
+): string {
+  if (outline.kind != null) {
+    return outline.kind;
+  } else if (outline.tokenizedText != null) {
```

Comment:
aren't the rest of these cases going to duplicate the content anyway?

I'd just use `kind` if it's available and not offer a tooltip otherwise.

Reply:
@wbinnssmith the `kind` variable is `undefined` every time , that's why I added other if conditions